docs/pages/upgrading/client-tools-autoupdate.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Teleport Client Tools Automatic Updates
+description: Explains how to use Teleport client tools (`tsh` and `tctl`) auto-updates.
+---
+
+This documentation explains how to keep Teleport client tools like `tsh` and `tctl` up-to-date.
+Updates can be automatic or self-managed, ensuring tools are secure, free from bugs,
+and compatible with your Teleport cluster.
```

Comment:
I just tried this feature and it's very nice. 

Can we add a note / section that outlines which version support this. e.g. if it's shipped with v15, v16, and v17. What's version I need to upgrade to get this feature.

Reply:
Looks like it was back-ported to Teleport 15.4.24,  I can't easily tell which other version it's in.